qdrant-landing/content/documentation/qdrant-alternatives.md

```diff
@@ -0,0 +1,78 @@
+| N/A            | **Scroll**     | Pinecone does not offer a way to iterate through all the vectors in a particular index. Qdrant has a `scroll` method to get them all without using search.                                                                                                                     |
+
+## Known limitations
+
+1. Pinecone does not support arbitrary JSON metadata, but a flat structure with strings, numbers, booleans, or lists of strings used as values. Qdrant accepts any JSON object as a payload, even nested structures.
+2. NULL values are not supported in Pinecone metadata but are handled properly by Qdrant.
+3. The maximum size of Pinecone metadata is 40kb per vector. 
+4. Pinecone, unlike Qdrant, does not support geolocation and filtering based on geographical criteria.
+5. Qdrant allows storing multiple vectors per point, and those might be of a different dimensionality. Pinecone doesn't support anything similar.
+6. Vectors in Pinecone are mandatory for each point. Qdrant supports optional vectors.
+
+It is worth mentioning, that **Pinecone will automatically create metadata indexes for all the fields**. Qdrant assumes you know
+your data and your future queries best, so it's up to you to choose the fields to be indexed. Thus, **you need to explicitly define the payload indexes while using Qdrant**.
+
+## Supported technologies
+
+Both tools support various programming languages providing official SDKs.
+
+|                           | Pinecone             | Qdrant               |
+|---------------------------|----------------------|----------------------|
+| **Python**                | ✅                    | ✅                    |
+| **JavaScript/TypeScript** | ✅                    | ✅                    |
+| **Rust**                  | ❌                    | ✅                    |
+| **Go**                    | ❌                    | ✅                    |
+
+There are also various community-driven projects aimed to provide the support for the other languages, but those are not officially 
+maintained, thus not mentioned here. However, it is still possible to interact with both engines through the HTTP REST or gRPC API. 
+That makes it easy to integrate with any technology of your choice.
+
+If you are a Python user, then both tools are well-integrated with the most popular libraries like LangChain, LlamaIndex, Haystack, and more. 
+Using any of those libraries makes it easier to experiment with different vector databases, as the transition should be seamless.
+
+## Comparison to Qdrant Cloud
+
+Both Pinecone and Qdrant Cloud offer a free cloud tier to check out the services, even without providing credit card details.
```

Comment:
Cost — even the Qdrant starter cloud ($25) is about 1/3rd the cost of Pinecone ($70)

Reply:
That is okay! Most of these docs get 80-90% of their readers in first 2-4 weeks. Since this is FOSS too, someone can flag and we can update.

Reply:
I'm not convinced that the rule applies to that comparison. The section is for all the users with experience with the other engines, and we hope there will be more and more of those who prefer switching to Qdrant. However, I agree with @davidmyriel - prices may vary, and we're also comparing different setups. Pinecone claims to support even 5M vectors in the lowest plan. To achieve the same with Qdrant, we would need to spend some more if our calculator is correct.

![image](https://github.com/qdrant/landing_page/assets/2649301/3f60f01c-524f-42a9-914c-61e40796ca7c)

![image](https://github.com/qdrant/landing_page/assets/2649301/df664255-1fe2-4b49-ab2e-0f21cb065b19)

Reply:
i recommend we keep it simple for now....lets just publish a minimal doc, add more later